docs/installation.md

```diff
@@ -22,12 +22,32 @@ scvi-tools can be installed via conda or pip. If you don't know which to choose,
 
 ### pip prerequisites
 
+0. If using conda/mamba, then just run `conda install -c anaconda pip` and skip this section.
 1. Install [Python], we prefer the [pyenv](https://github.com/pyenv/pyenv/) version management system, along with [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv/).
-2. Install [PyTorch]. If you have an Nvidia GPU, be sure to install a version of PyTorch that supports it -- scvi-tools runs much faster with a discrete GPU.
+2. Install [PyTorch] and [jax]. If you have an Nvidia GPU, be sure to install versions of PyTorch and jax that support it -- scvi-tools runs much faster with a discrete GPU.
 
-:::{note}
-Installing scvi-tools on a Mac with Apple Silicon is only possible using a native version of Python. A native version of Python can be installed with an Apple Silicon version of miniconda (which can be installed from a native version of homebrew). This is due to an scvi-tools dependency on jax, which cannot be run via Rosetta.
-:::
+### Apple silicon prerequisites
+
+Installing scvi-tools on a Mac with Apple Silicon is only possible using a native version of Python. A native version of Python can be installed with an Apple Silicon version of mambaforge (which can be installed from a native version of homebrew via `brew install --cask mambaforge`). This is due to an scvi-tools dependency on jax, which cannot be run via Rosetta.
```

Comment:
Should we also note here that PyTorch is not fully supported yet with metal backend?